chromium_src/chrome/browser/ui/views/bubble/bubble_contents_wrapper.h

```diff
@@ -24,9 +24,8 @@
                       WindowOpenDisposition disposition,                   \
                       const blink::mojom::WindowFeatures& window_features, \
                       bool user_gesture, bool* was_blocked) override;      \
-  std::vector<int32_t>& GetPopupIds() {                                    \
-    return popup_ids_;                                                     \
-  }                                                                        \
+  const std::vector<int32_t>& popup_ids() { return popup_ids_; }           \
```

Comment:
`popup_ids() const`

Reply:
Fixed in 9501de7a282bd7b5f38697159793ee3a856db1eb.